AuroraAssetEditor/Classes/XboxAssetDownloader.cs

```diff
@@ -52,24 +53,21 @@ internal class XboxAssetDownloader {
             try {
             var ret = new List<XboxLocale>();
             var tmp = new List<string>();
-            var wc = new WebClient();
-            var data = Encoding.UTF8.GetString(wc.DownloadData("http://www.xbox.com/Shell/ChangeLocale")).Split('>');
-            for(var i = 0; i < data.Length; i++) {
-                if(!data[i].ToLower().Contains("?targetlocale="))
-                    continue;
-                var index = data[i].ToLower().IndexOf("?targetlocale=", StringComparison.Ordinal) + 14;
-                var id = data[i].Substring(index);
-                index = id.IndexOf('"');
-                if(index <= 0)
-                    continue;
-                id = id.Substring(0, index);
-                if(tmp.Contains(id))
-                    continue;
-                var name = data[i + 1];
-                name = name.Substring(0, name.IndexOf("</a", StringComparison.Ordinal));
-                name = HttpUtility.HtmlDecode(name);
-                ret.Add(new XboxLocale(id, name));
-                tmp.Add(id);
+            var wc = new WebClient();
+            // Enable TLS 1.0 (0xC0), TLS 1.1 (0x300), and TLS 1.2 (0xC00) security protocols
+            // https://docs.microsoft.com/en-us/dotnet/api/system.net.securityprotocoltype?view=net-6.0
+            System.Net.ServicePointManager.SecurityProtocol |= (SecurityProtocolType)(0xc0 | 0x300 | 0xc00);

```

Comment:
I tried to use `... |= SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12` but I am getting an error that `SecurityProtocolType.Tls11` and `SecurityProtocolType.Tls12` are not defined. But it works when manually creating a flag

Reply:
Decided to look into this again today and it appears that xbox.com only supports TLS 1.2 and TLS 1.3.
I will fixup this commit so that TLS 1.0 and TLS 1.1 are no longer enabled as well as create a new commit to bump the version of the app to 1.3.2.0